openedx/core/apidocs.py

```diff
@@ -44,7 +44,7 @@ def dedent(text):
     return textwrap.dedent(text)
 
 
-def swagger_auto_schema(**kwargs):
+def schema(*args, **kwargs):
```

Comment:
Please change the docstring, because it now does more than dedenting description fields.

Also, if we are serious about hiding the fact that it's drf_yasg underneath, then `schema` should explicitly take a subset of swagger_auto_schema's arguments (renamed if appropriate) and then pass those to swagger_auto_schema.

By accepting `**kwargs` and passing them through to swagger_auto_schema, we essentially commit to supporting all of that decorator's functionality, making it much tougher to switch to another API generation scheme.

Reply:
Agreed -- but we will lose the ability to call the `@schema` decorator with no argument and parenthesis.